src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -155,99 +169,100 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
                 return '';
             }
 
-            var defaultAttribute = mediaSource.DefaultSubtitleStreamIndex === t.Index ? ' default' : '';
+            const defaultAttribute = mediaSource.DefaultSubtitleStreamIndex === t.Index ? ' default' : '';
 
-            var language = t.Language || 'und';
-            var label = t.Language || 'und';
+            const language = t.Language || 'und';
+            const label = t.Language || 'und';
             return '<track id="textTrack' + t.Index + '" label="' + label + '" kind="subtitles" src="' + getTextTrackUrl(t, item) + '" srclang="' + language + '"' + defaultAttribute + '></track>';
 
         }).join('');
     }
 
     function getDefaultProfile() {
+        return import('browserdeviceprofile').then(profileBuilder => {
 
-        return new Promise(function (resolve, reject) {
-
-            require(['browserdeviceprofile'], function (profileBuilder) {
-
-                resolve(profileBuilder({}));
-            });
+            return profileBuilder({});
         });
     }
 
-    function HtmlVideoPlayer() {
-
-        if (browser.edgeUwp) {
-            this.name = 'Windows Video Player';
-        } else {
-            this.name = 'Html Video Player';
-        }
-
-        this.type = 'mediaplayer';
-        this.id = 'htmlvideoplayer';
-
-        // Let any players created by plugins take priority
-        this.priority = 1;
-
-        var videoDialog;
-
-        var winJsPlaybackItem;
-
-        var subtitleTrackIndexToSetOnPlaying;
-        var audioTrackIndexToSetOnPlaying;
-
-        var lastCustomTrackMs = 0;
-        var currentClock;
-        var currentSubtitlesOctopus;
-        var currentAssRenderer;
-        var customTrackIndex = -1;
-
-        var showTrackOffset;
-        var currentTrackOffset;
+    /**
+     * Private:
```

Comment:
No sure this is needed, since you can use `#privateProperties` now, plus the better way to do it would be using JSDoc's `@private` tag.

Reply:
Yeah JSDoc didn't work with without setting those to a value but now with private properties this might work